project/src/main/background.gd

```diff
@@ -60,18 +59,18 @@ func change(immediate: bool = false) -> void:
 	else:
 		textures[0].modulate = to_transparent(texture_color)
 		
-		_change_tween.interpolate_property(_color_rect, "color",
-				_color_rect.color, rect_color, TWEEN_DURATION)
-		_change_tween.interpolate_property(textures[0], "modulate",
-				textures[0].modulate, texture_color, TWEEN_DURATION)
-		_change_tween.interpolate_property(previous_texture, "modulate",
-				previous_texture.modulate, to_transparent(previous_texture.modulate), TWEEN_DURATION)
-		_change_tween.start()
+		var change_tween := create_tween().set_parallel()
+		change_tween.tween_property(_color_rect, "color",
+				rect_color, TWEEN_DURATION)
+		change_tween.tween_property(textures[0], "modulate",
+				texture_color, TWEEN_DURATION)
+		change_tween.tween_property(previous_texture, "modulate",
+				to_transparent(previous_texture.modulate), TWEEN_DURATION)
+		change_tween.chain().tween_callback(self, "_on_ChangeTween_tween_completed", [previous_texture])
 
 
-func _on_ChangeTween_tween_completed(object: Object, key: NodePath) -> void:
-	if key == ":modulate" and object.modulate.a == 0.0:
-		object.visible = false
+func _on_ChangeTween_tween_completed(previous_texture: TextureRect) -> void:
+	previous_texture.visible = false
```

Comment:
The tween callback could just set the previous texture visible to false; this method is not necessary.